src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+}
+
+/* #region image */
+.image {
+  margin: 0;
+  position: absolute;
+  height: 19px;
+  color: #687c94;
+}
+
+.big__image {
+  top: 25px;
+  left: 26px;
+}
+
+.small__image {
+  top: 15px;
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
+/* #endregion */
+
+/* #region big__input */
+.input {
+  display: block;
+  position: relative;
+  margin-top: 20px;
+  width: 100%;
+  background-color: var(--background-color);
+}
+
+.big__input {
+  height: 70px;
+  line-height: 70px;
+  padding-left: 62px;
+
+  font-family: inherit;
+  background-color: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px #3d4e611a;
+}
+
+.big__input::placeholder {
+  font-size: 16px;
+  font-family: inherit;
+  height: 22px;
+  line-height: 22px;
+}
+
+.big__input:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+}
+
+.big__input:focus {
+  font-family: inherit;
+  font-weight: 700;
+
+  background: linear-gradient(rgba(255, 255, 255, 1), rgba(246, 246, 247, 1));
+  border-radius: 3px;
+  outline: none;
+  border: 1px solid var(--border-color);
+  text-shadow: 0 4px 4px #00000040;
+}
+/* #endregion */
+
+/* #region small_input */
+.small__input {
+  height: 42px;
+  line-height: 42px;
+  padding-left: 33px;
+
+  font-family: inherit;
+  font-size: 14px;
+  background-color: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px #3d4e611a;
```

Comment:
You use same styles for the big input 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/96eedbfe-ec4e-402f-9c99-3c687853fccf)